com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/util/StreamUtil.java

```diff
@@ -20,23 +20,23 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.sap.cloud.lm.sl.cf.client.message.Messages;
+import com.sap.cloud.lm.sl.common.ContentException;
+
 public class StreamUtil {

```

Comment:
How this is different from the StreamManager?

Reply:
This class has only static methods. StreamManager has constructor with parameter InputStream. This is the reason to split this StreamUtil class.